rs-matter/src/data_model/subscriptions.rs

```diff
@@ -0,0 +1,142 @@
+/*
+ *
+ *    Copyright (c) 2020-2022 Project CHIP Authors
```

Comment:
supernit: for a new file we could say 2024

Reply:
Hmmm, is this what is practiced in the `connectedhomeip` repo? I would've imagined you guys just run a regep from time to time that updates this everywhere...

Reply:
We generally try to not update files as this seems just churn. We had proposals from some to create an `automated workflow` however even that seems churn, so we asked if any lawyer really insists on this and nobody insisted as a requirement.

So overall we try new files to be correct and then they tend to not be updated (however we will not reject an update).